projects/storefrontlib/src/cms-components/asm/services/asm-enabler.service.ts

```diff
@@ -18,14 +18,12 @@ import { AsmMainUiComponent } from '../asm-main-ui/asm-main-ui.component';
   providedIn: 'root',
 })
 export class AsmEnablerService {
-  /** indicates whether the ASM UI has been added already */
-  private isUiAdded = false;
-
   constructor(
     protected location: Location,
     protected winRef: WindowRef,
     protected componentFactoryResolver: ComponentFactoryResolver,
```

Comment:
The componentFactoryResolver is obsolete now. 
We also need add migration for the constructor in the schematics project.